src/header/map.rs

```diff
@@ -1,3 +1,12 @@
+// Copyright 2017-2021 http-rs authors
```

Comment:
I think "http-rs" is actually a separate organization. Maybe we just leave off about the rest of the copyright, assuming the top level license works, and we just say "Parts of this file are ..."?

Reply:
Ok. http-rs authors is exactly what the top-level license files write too.